### PR TITLE
Testing workflow changes and commit SHAs

### PR DESCRIPTION
**Pull request by Anirban166**

### CODE REVIEW

**Comment by tdhock** [comment]
I see that Fast is constant time and others are linear (same as slow), is that normal? I was expecting all to be linear and others to be same as fast (not slow).

**Comment by Anirban166** [comment]
> I see that Fast is constant time and others are linear (same as slow), is that normal? I was expecting all to be linear and others to be same as fast (not slow).

It's because #5427 hasn't been merged - once it does, I'd expect at least 'Base', 'merge-base', and 'CRAN' to be aligned with 'Fast' (also 'HEAD' if the PR does not affect or degrade performance for that test)

**Comment by Anirban166** [comment]
> I see that Fast is constant time and others are linear (same as slow), is that normal? I was expecting all to be linear and others to be same as fast (not slow).

It's because #5427 hasn't been merged or the current master branch is slow - once it does, I'd expect at least 'Base', 'merge-base', and 'CRAN' to be aligned with 'Fast' (also 'HEAD' if the PR does not affect or degrade performance for that test)

**Comment by Anirban166** [comment]
> I see that Fast is constant time and others are linear (same as slow), is that normal? I was expecting all to be linear and others to be same as fast (not slow).

It's because the current master branch is indeed slow since #5427 hasn't been merged yet - once that is done, I'd expect at least 'Base', 'merge-base', and 'CRAN' to be aligned with 'Fast' (also 'HEAD' if the triggering PR does not affect or degrade performance for that test)

File: .ci/atime/tests.R [hunk 1]
```diff
@@ -129,9 +129,10 @@ test.list <- list(
   pkg.edit.fun = pkg.edit.fun,     
   N = 10^seq(1, 7),
   setup = quote({ 
-    DT = replicate(N, 1, simplify = FALSE)
+    L <- replicate(N, 1, simplify = FALSE)
+    data.table:::setattr(L, "class", NULL)
```
Comment (tdhock) [inline .ci/atime/tests.R]: setattr should be moved inside expr, please see https://github.com/Rdatatable/data.table/pull/5427#issuecomment-2073368617

Comment (Anirban166) [reply .ci/atime/tests.R]: Oops yes you're right

File: .ci/atime/tests.R [hunk 2]
```diff
@@ -130,9 +130,7 @@ test.list <- list(
   N = 10^seq(1, 7),
   setup = quote({ 
     L <- replicate(N, 1, simplify = FALSE)
-    data.table:::setattr(L, "class", NULL)    
-    data.table:::setDT(L)  
   }),
-  expr = quote(data.table:::setDT(DT)),
+  expr = quote(data.table:::setattr(L, "class", NULL); data.table:::setDT(DT)),
```
Comment (tdhock) [inline .ci/atime/tests.R]: please use {} to break over two lines instead of using ;
also it should be L not DT

**Pull request closed without merging**